<!DOCTYPE html
PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
<html>
<head>
<title>BRESEQ :: Marginal Predictions</title>
<style type = "text/css">
body {font-family: sans-serif; font-size: 11pt;}
th {background-color: rgb(0,0,0); color: rgb(255,255,255);}
table {background-color: rgb(1,0,0); color: rgb(0,0,0);}
tr {background-color: rgb(255,255,255);}
.mutation_in_codon {color:red; text-decoration : underline;}
.snp_type_synonymous{color:green;}
.snp_type_nonsynonymous{color:blue;}
.snp_type_nonsense{color:red;}
.mutation_header_row {background-color: rgb(0,130,0);}
.read_alignment_header_row {background-color: rgb(255,0,0);}
.missing_coverage_header_row {background-color: rgb(0,100,100);}
.new_junction_header_row {background-color: rgb(0,0,155);}
.copy_number_header_row {background-color: rgb(153,102,0);}
.alternate_table_row_0 {background-color: rgb(255,255,255);}
.alternate_table_row_1 {background-color: rgb(235,235,235);}
.gray_table_row {background-color: rgb(230,230,245);}
.polymorphism_table_row {background-color: rgb(160,255,160);}
.highlight_table_row {background-color: rgb(192,255,255);}
.reject_table_row {background-color: rgb(255,200,165);}
.user_defined_table_row {background-color: rgb(255,255,0);}
.information_table_row {background-color: rgb(200,255,255);}
.junction_repeat {background-color: rgb(255,165,0)}
.junction_gene {}
.hidden { display: none; }
.unhidden { display: block; }

</style>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1" />
<script type="text/javascript">
  function hideTog(divID) {
    var item = document.getElementById(divID);
    if (item) {
      item.className=(item.className=='hidden')?'unhidden':'hidden';
    }
  }
  function showTog(butID) {
    var button = document.getElementById(butID);
    if (button) {
      button.value=(button.value=='Show')?'Hide':'Show';
    }
  }
</script>

</head>
<body>
<table width="100%" border="0" cellspacing="0" cellpadding="3">
<tr>
<td><a href="http://barricklab.org/breseq"><img src="evidence/breseq_small.png" /></a></td>
<td width="100%">
<b><i>breseq</i></b>&nbsp;&nbsp;version 0.38.1&nbsp;&nbsp;
<br><a href="index.html">mutation predictions</a> | 
<a href="marginal.html">marginal predictions</a> | 
<a href="summary.html">summary statistics</a> | 
<a href="output.gd">genome diff</a> | 
<a href="log.txt">command line log</a>
</td></tr></table>

<p>
<p>
<table border="0" cellspacing="1", cellpadding="3">
<tr><th colspan="12" align="left" class="read_alignment_header_row">Marginal read alignment evidence (highest frequency 20 of 49 shown, sorted by frequency from high to low)</th></tr>
<tr>
<th>&nbsp;</th>
<th>seq&nbsp;id</th>
<th colspan="2">position</th>
<th>ref</th>
<th>new</th>
<th>freq</th>
<th>score&nbsp;(cons/poly)</th>
<th>reads</th>
<th>annotation</th>
<th>genes</th>
<th width="100%">product</th>
</tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_36.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">1,611,384</td><td align="right">0</td><td align="center">G</td><td align="center">C</td><td align="right">63.0%
</td><td align="center">59.0
/&nbsp;192.2
</td><td align="center">145</td><td align="center"><font class="snp_type_nonsynonymous">V152L</font>&nbsp;(<font class="mutation_in_codon">G</font>TC&rarr;<font class="mutation_in_codon">C</font>TC)&nbsp;&Dagger;</td><td align="center"><i>ACX60_RS07530</i></td><td align="left">hypothetical protein</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_45.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">2,204,395</td><td align="right">0</td><td align="center">C</td><td align="center">G</td><td align="right">42.9%
</td><td align="center">122.7
/&nbsp;80.5
</td><td align="center">97</td><td align="center">intergenic&nbsp;(&#8209;7/&#8209;252)</td><td align="center"><i>ACX60_RS10510/ACX60_RS10515</i></td><td align="left">hypothetical protein/hypothetical protein</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_46.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">2,204,396</td><td align="right">0</td><td align="center">C</td><td align="center">G</td><td align="right">41.9%
</td><td align="center">115.4
/&nbsp;74.9
</td><td align="center">94</td><td align="center">intergenic&nbsp;(&#8209;8/&#8209;251)</td><td align="center"><i>ACX60_RS10510/ACX60_RS10515</i></td><td align="left">hypothetical protein/hypothetical protein</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_37.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">1,611,385</td><td align="right">0</td><td align="center">T</td><td align="center">C</td><td align="right">41.4%
</td><td align="center">120.3
/&nbsp;114.3
</td><td align="center">152</td><td align="center"><font class="snp_type_nonsynonymous">V152A</font>&nbsp;(G<font class="mutation_in_codon">T</font>C&rarr;G<font class="mutation_in_codon">C</font>C)&nbsp;&Dagger;</td><td align="center"><i>ACX60_RS07530</i></td><td align="left">hypothetical protein</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_61.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">3,632,436</td><td align="right">0</td><td align="center">A</td><td align="center">C</td><td align="right">39.4%
</td><td align="center">236.8
/&nbsp;38.0
</td><td align="center">142</td><td align="center"><font class="snp_type_nonsynonymous">L70F</font>&nbsp;(TT<font class="mutation_in_codon">A</font>&rarr;TT<font class="mutation_in_codon">C</font>)&nbsp;</td><td align="center"><i>ACX60_RS17130</i></td><td align="left">MFS transporter</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_48.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">2,349,705</td><td align="right">0</td><td align="center">T</td><td align="center">C</td><td align="right">39.3%
</td><td align="center">101.0
/&nbsp;68.6
</td><td align="center">112</td><td align="center">intergenic&nbsp;(+101/&#8209;5)</td><td align="center"><i>ACX60_RS11215/ACX60_RS11220</i></td><td align="left">hypothetical protein/hypothetical protein</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_44.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">2,204,394</td><td align="right">0</td><td align="center">A</td><td align="center">G</td><td align="right">38.4%
</td><td align="center">114.2
/&nbsp;81.2
</td><td align="center">99</td><td align="center">intergenic&nbsp;(&#8209;6/&#8209;253)</td><td align="center"><i>ACX60_RS10510/ACX60_RS10515</i></td><td align="left">hypothetical protein/hypothetical protein</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_47.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">2,204,397</td><td align="right">0</td><td align="center">C</td><td align="center">G</td><td align="right">38.1%
</td><td align="center">139.5
/&nbsp;57.3
</td><td align="center">94</td><td align="center">intergenic&nbsp;(&#8209;9/&#8209;250)</td><td align="center"><i>ACX60_RS10510/ACX60_RS10515</i></td><td align="left">hypothetical protein/hypothetical protein</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_52.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">2,736,592</td><td align="right">0</td><td align="center">C</td><td align="center">G</td><td align="right">35.2%
</td><td align="center">245.0
/&nbsp;77.8
</td><td align="center">133</td><td align="center"><font class="snp_type_nonsynonymous">S479T</font>&nbsp;(A<font class="mutation_in_codon">G</font>T&rarr;A<font class="mutation_in_codon">C</font>T)&nbsp;</td><td align="center"><i>fadH</i></td><td align="left">NADPH&#8209;dependent 2,4&#8209;dienoyl&#8209;CoA reductase</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_59.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">3,125,709</td><td align="right">0</td><td align="center">C</td><td align="center">G</td><td align="right">33.9%
</td><td align="center">191.8
/&nbsp;76.2
</td><td align="center">126</td><td align="center"><font class="snp_type_nonsynonymous">G330R</font>&nbsp;(<font class="mutation_in_codon">G</font>GC&rarr;<font class="mutation_in_codon">C</font>GC)&nbsp;</td><td align="center"><i>ACX60_RS14780</i></td><td align="left">acyl&#8209;CoA dehydrogenase</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_55.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">2,984,854</td><td align="right">0</td><td align="center">G</td><td align="center">C</td><td align="right">33.9%
</td><td align="center">259.8
/&nbsp;76.2
</td><td align="center">131</td><td align="center">noncoding&nbsp;(348/355&nbsp;nt)</td><td align="center"><i>ACX60_RS19000</i></td><td align="left">bacterial RNase P</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_22.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">682,086</td><td align="right">0</td><td align="center">A</td><td align="center">C</td><td align="right">33.3%
</td><td align="center">314.3
/&nbsp;17.9
</td><td align="center">138</td><td align="center">intergenic&nbsp;(+50/+75)</td><td align="center"><i>ACX60_RS03295/ACX60_RS03300</i></td><td align="left">membrane protein/hypothetical protein</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_41.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">1,939,477</td><td align="right">0</td><td align="center">T</td><td align="center">A</td><td align="right">33.3%
</td><td align="center">57.8
/&nbsp;50.0
</td><td align="center">36</td><td align="center">pseudogene&nbsp;(6/369&nbsp;nt)</td><td align="center"><i>antC</i></td><td align="left">anthranilate dioxygenase reductase</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_56.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">2,984,858</td><td align="right">0</td><td align="center">G</td><td align="center">C</td><td align="right">33.1%
</td><td align="center">247.5
/&nbsp;77.3
</td><td align="center">129</td><td align="center">noncoding&nbsp;(344/355&nbsp;nt)</td><td align="center"><i>ACX60_RS19000</i></td><td align="left">bacterial RNase P</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_21.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">607,375</td><td align="right">0</td><td align="center">T</td><td align="center">C</td><td align="right">33.0%
</td><td align="center">156.7
/&nbsp;36.3
</td><td align="center">94</td><td align="center"><font class="snp_type_synonymous">D178D</font>&nbsp;(GA<font class="mutation_in_codon">T</font>&rarr;GA<font class="mutation_in_codon">C</font>)&nbsp;</td><td align="center"><i>ACX60_RS02965</i></td><td align="left">hypothetical protein</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_34.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">1,529,192</td><td align="right">0</td><td align="center">A</td><td align="center">C</td><td align="right">31.7%
</td><td align="center">250.2
/&nbsp;20.2
</td><td align="center">124</td><td align="center">intergenic&nbsp;(+51/&#8209;588)</td><td align="center"><i>ACX60_RS07100/ACX60_RS07105</i></td><td align="left">aminopeptidase N/spore Coat protein U domain family</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_20.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">460,996</td><td align="right">0</td><td align="center">G</td><td align="center">C</td><td align="right">30.5%
</td><td align="center">308.7
/&nbsp;68.5
</td><td align="center">146</td><td align="center"><font class="snp_type_nonsynonymous">K233N</font>&nbsp;(AA<font class="mutation_in_codon">G</font>&rarr;AA<font class="mutation_in_codon">C</font>)&nbsp;</td><td align="center"><i>ACX60_RS02250</i></td><td align="left">hypothetical protein</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_29.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">984,391</td><td align="right">0</td><td align="center">T</td><td align="center">G</td><td align="right">30.1%
</td><td align="center">302.6
/&nbsp;15.9
</td><td align="center">105</td><td align="center"><font class="snp_type_nonsynonymous">K297Q</font>&nbsp;(<font class="mutation_in_codon">A</font>AG&rarr;<font class="mutation_in_codon">C</font>AG)&nbsp;</td><td align="center"><i>ACX60_RS04655</i></td><td align="left">AraC family transcriptional regulator</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_57.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">3,121,854</td><td align="right">0</td><td align="center">A</td><td align="center">C</td><td align="right">29.8%
</td><td align="center">346.9
/&nbsp;13.7
</td><td align="center">131</td><td align="center"><font class="snp_type_nonsynonymous">Y148S</font>&nbsp;(T<font class="mutation_in_codon">A</font>T&rarr;T<font class="mutation_in_codon">C</font>T)&nbsp;</td><td align="center"><i>tcuB</i></td><td align="left">tricarballylate utilization protein B</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_10.html">*</a></td><td align="center">CP000522</td><td align="right">8,237</td><td align="right">0</td><td align="center">A</td><td align="center">C</td><td align="right">28.9%
</td><td align="center">315.1
/&nbsp;10.2
</td><td align="center">123</td><td align="center">intergenic&nbsp;(&#8209;552/+365)</td><td align="center"><i>A1S_3466/A1S_3467</i></td><td align="left">Resolvase/hypothetical protein</td></tr>
</table>

<p>
<table border="0" cellspacing="1" cellpadding="3">
<tr><th colspan="12" align="left" class="new_junction_header_row">Marginal new junction evidence (sorted from low to high skew)</th></tr>
<!-- Header Lines for New Junction -->
<tr>
<th colspan="2">&nbsp;</th>
<th>seq&nbsp;id</th>
<th>position</th>
<th>reads&nbsp;(cov)</th>
<th>reads&nbsp;(cov)</th>
<th>score</th>
<th>skew</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">product</th>
</tr>

<!-- Item Lines for New Junction -->
<!-- Side 1 Item Lines for New Junction -->
<tr class="mutation_table_row_0">
<td rowspan="2"><a href="evidence/JC_70.html">*</a></td>
<td rowspan="1"><a href="evidence/JC_SIDE_1_70.html">?</a></td>
<td rowspan="1" class="junction_gene">CP000522</td>
<td align="center" class="junction_gene">=&nbsp;11251</td><td align="center" class="junction_gene">142 (0.850)</td><td rowspan="2" align="center">9 (0.040)</td>
<td rowspan="2" align="center">8/258</td>
<td rowspan="2" align="center">8.1</td>
<td rowspan="2" align="center">4.9%</td>
<td align="center" class="junction_gene">intergenic&nbsp;(&#8209;528/+825)</td>
<td align="center" class="junction_gene"><i>A1S_3469/A1S_3470</i></td>
<td class="junction_gene">Diaminopimelate decarboxylase/regulatory protein LysR</td>
</tr>
<!-- Side 2 Item Lines for New Junction -->
<tr class="mutation_table_row_0">
<td rowspan="1"><a href="evidence/JC_SIDE_2_70.html">?</a></td><td rowspan="1" class="junction_gene">CP000523</td>
<td align="center" class="junction_gene">=&nbsp;5242</td>
<td align="center" class="junction_gene">226 (0.890)</td><td align="center" class="junction_gene">intergenic&nbsp;(+109/&#8209;955)</td>
<td align="center" class="junction_gene"><i>A1S_3474/A1S_3475</i></td>
<td class="junction_gene">hypothetical protein/hypothetical protein</td>
</tr>
<!-- Side 1 Item Lines for New Junction -->
<tr class="mutation_table_row_1">
<td rowspan="2"><a href="evidence/JC_69.html">*</a></td>
<td rowspan="1"><a href="evidence/JC_SIDE_1_69.html">?</a></td>
<td rowspan="1" class="junction_gene">CP000522</td>
<td align="center" class="junction_gene">11236&nbsp;=</td><td align="center" class="junction_gene">140 (0.840)</td><td rowspan="2" align="center">11 (0.050)</td>
<td rowspan="2" align="center">8/258</td>
<td rowspan="2" align="center">8.1</td>
<td rowspan="2" align="center">5.9%</td>
<td align="center" class="junction_gene">intergenic&nbsp;(&#8209;513/+840)</td>
<td align="center" class="junction_gene"><i>A1S_3469/A1S_3470</i></td>
<td class="junction_gene">Diaminopimelate decarboxylase/regulatory protein LysR</td>
</tr>
<!-- Side 2 Item Lines for New Junction -->
<tr class="mutation_table_row_1">
<td rowspan="1"><a href="evidence/JC_SIDE_2_69.html">?</a></td><td rowspan="1" class="junction_gene">CP000523</td>
<td align="center" class="junction_gene">5259&nbsp;=</td>
<td align="center" class="junction_gene">226 (0.890)</td><td align="center" class="junction_gene">intergenic&nbsp;(+126/&#8209;938)</td>
<td align="center" class="junction_gene"><i>A1S_3474/A1S_3475</i></td>
<td class="junction_gene">hypothetical protein/hypothetical protein</td>
</tr>
<!-- Side 1 Item Lines for New Junction -->
<tr class="mutation_table_row_0">
<td rowspan="2"><a href="evidence/JC_75.html">*</a></td>
<td rowspan="1"><a href="evidence/JC_SIDE_1_75.html">?</a></td>
<td rowspan="1" class="junction_gene">NZ_CP012004</td>
<td align="center" class="junction_gene">900293&nbsp;=</td><td align="center" class="junction_gene">145 (1.050)</td><td rowspan="2" align="center">3 (0.020)</td>
<td rowspan="2" align="center">3/268</td>
<td rowspan="2" align="center">12.1</td>
<td rowspan="2" align="center">2.2%</td>
<td align="center" class="junction_gene">coding&nbsp;(2761/3999&nbsp;nt)</td>
<td align="center" class="junction_gene"><i>ACX60_RS04270</i></td>
<td class="junction_gene">peptide synthetase</td>
</tr>
<!-- Side 2 Item Lines for New Junction -->
<tr class="mutation_table_row_0">
<td rowspan="1"><a href="evidence/JC_SIDE_2_75.html">?</a></td><td rowspan="1" class="junction_gene">NZ_CP012004</td>
<td align="center" class="junction_gene">=&nbsp;900289</td>
<td align="center" class="junction_gene">132 (1.030)</td><td align="center" class="junction_gene">coding&nbsp;(2757/3999&nbsp;nt)</td>
<td align="center" class="junction_gene"><i>ACX60_RS04270</i></td>
<td class="junction_gene">peptide synthetase</td>
</tr>
</table>
</body></html>