<!DOCTYPE html
PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
<html>
<head>
<title>BRESEQ :: Marginal Predictions</title>
<style type = "text/css">
body {font-family: sans-serif; font-size: 11pt;}
th {background-color: rgb(0,0,0); color: rgb(255,255,255);}
table {background-color: rgb(1,0,0); color: rgb(0,0,0);}
tr {background-color: rgb(255,255,255);}
.mutation_in_codon {color:red; text-decoration : underline;}
.snp_type_synonymous{color:green;}
.snp_type_nonsynonymous{color:blue;}
.snp_type_nonsense{color:red;}
.mutation_header_row {background-color: rgb(0,130,0);}
.read_alignment_header_row {background-color: rgb(255,0,0);}
.missing_coverage_header_row {background-color: rgb(0,100,100);}
.new_junction_header_row {background-color: rgb(0,0,155);}
.copy_number_header_row {background-color: rgb(153,102,0);}
.alternate_table_row_0 {background-color: rgb(255,255,255);}
.alternate_table_row_1 {background-color: rgb(235,235,235);}
.gray_table_row {background-color: rgb(230,230,245);}
.polymorphism_table_row {background-color: rgb(160,255,160);}
.highlight_table_row {background-color: rgb(192,255,255);}
.reject_table_row {background-color: rgb(255,200,165);}
.user_defined_table_row {background-color: rgb(255,255,0);}
.information_table_row {background-color: rgb(200,255,255);}
.junction_repeat {background-color: rgb(255,165,0)}
.junction_gene {}
.hidden { display: none; }
.unhidden { display: block; }

</style>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1" />
<script type="text/javascript">
  function hideTog(divID) {
    var item = document.getElementById(divID);
    if (item) {
      item.className=(item.className=='hidden')?'unhidden':'hidden';
    }
  }
  function showTog(butID) {
    var button = document.getElementById(butID);
    if (button) {
      button.value=(button.value=='Show')?'Hide':'Show';
    }
  }
</script>

</head>
<body>
<table width="100%" border="0" cellspacing="0" cellpadding="3">
<tr>
<td><a href="http://barricklab.org/breseq"><img src="evidence/breseq_small.png" /></a></td>
<td width="100%">
<b><i>breseq</i></b>&nbsp;&nbsp;version 0.38.1&nbsp;&nbsp;
<br><a href="index.html">mutation predictions</a> | 
<a href="marginal.html">marginal predictions</a> | 
<a href="summary.html">summary statistics</a> | 
<a href="output.gd">genome diff</a> | 
<a href="log.txt">command line log</a>
</td></tr></table>

<p>
<p>
<table border="0" cellspacing="1", cellpadding="3">
<tr><th colspan="12" align="left" class="read_alignment_header_row">Marginal read alignment evidence (highest frequency 20 of 42 shown, sorted by frequency from high to low)</th></tr>
<tr>
<th>&nbsp;</th>
<th>seq&nbsp;id</th>
<th colspan="2">position</th>
<th>ref</th>
<th>new</th>
<th>freq</th>
<th>score&nbsp;(cons/poly)</th>
<th>reads</th>
<th>annotation</th>
<th>genes</th>
<th width="100%">product</th>
</tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_39.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">1,939,477</td><td align="right">0</td><td align="center">T</td><td align="center">A</td><td align="right">38.5%
</td><td align="center">10.2
/&nbsp;17.7
</td><td align="center">13</td><td align="center">pseudogene&nbsp;(6/369&nbsp;nt)</td><td align="center"><i>antC</i></td><td align="left">anthranilate dioxygenase reductase</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_36.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">1,587,008</td><td align="right">0</td><td align="center">A</td><td align="center">C</td><td align="right">35.8%
</td><td align="center">131.7
/&nbsp;17.0
</td><td align="center">82</td><td align="center"><font class="snp_type_nonsynonymous">H10Q</font>&nbsp;(CA<font class="mutation_in_codon">T</font>&rarr;CA<font class="mutation_in_codon">G</font>)&nbsp;</td><td align="center"><i>ACX60_RS07355</i></td><td align="left">FMN&#8209;binding glutamate synthase family protein</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_49.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">2,668,823</td><td align="right">0</td><td align="center">T</td><td align="center">G</td><td align="right">35.5%
</td><td align="center">188.5
/&nbsp;36.7
</td><td align="center">121</td><td align="center"><font class="snp_type_nonsynonymous">L216F</font>&nbsp;(TT<font class="mutation_in_codon">A</font>&rarr;TT<font class="mutation_in_codon">C</font>)&nbsp;</td><td align="center"><i>ACX60_RS12775</i></td><td align="left">GntR family transcriptional regulator</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_32.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">1,120,875</td><td align="right">0</td><td align="center">A</td><td align="center">C</td><td align="right">34.1%
</td><td align="center">189.5
/&nbsp;11.0
</td><td align="center">90</td><td align="center">intergenic&nbsp;(+40/+16)</td><td align="center"><i>ACX60_RS05265/ACX60_RS05270</i></td><td align="left">RNA polymerase&#8209;associated protein RapA/membrane protein</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_42.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">2,204,394</td><td align="right">0</td><td align="center">A</td><td align="center">G</td><td align="right">33.3%
</td><td align="center">89.6
/&nbsp;32.6
</td><td align="center">57</td><td align="center">intergenic&nbsp;(&#8209;6/&#8209;253)</td><td align="center"><i>ACX60_RS10510/ACX60_RS10515</i></td><td align="left">hypothetical protein/hypothetical protein</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_54.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">3,125,709</td><td align="right">0</td><td align="center">C</td><td align="center">G</td><td align="right">33.1%
</td><td align="center">242.0
/&nbsp;70.5
</td><td align="center">130</td><td align="center"><font class="snp_type_nonsynonymous">G330R</font>&nbsp;(<font class="mutation_in_codon">G</font>GC&rarr;<font class="mutation_in_codon">C</font>GC)&nbsp;</td><td align="center"><i>ACX60_RS14780</i></td><td align="left">acyl&#8209;CoA dehydrogenase</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_35.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">1,529,192</td><td align="right">0</td><td align="center">A</td><td align="center">C</td><td align="right">32.1%
</td><td align="center">175.0
/&nbsp;14.2
</td><td align="center">84</td><td align="center">intergenic&nbsp;(+51/&#8209;588)</td><td align="center"><i>ACX60_RS07100/ACX60_RS07105</i></td><td align="left">aminopeptidase N/spore Coat protein U domain family</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_56.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">3,632,436</td><td align="right">0</td><td align="center">A</td><td align="center">C</td><td align="right">31.6%
</td><td align="center">365.6
/&nbsp;45.4
</td><td align="center">190</td><td align="center"><font class="snp_type_nonsynonymous">L70F</font>&nbsp;(TT<font class="mutation_in_codon">A</font>&rarr;TT<font class="mutation_in_codon">C</font>)&nbsp;</td><td align="center"><i>ACX60_RS17130</i></td><td align="left">MFS transporter</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_10.html">*</a></td><td align="center">CP000522</td><td align="right">8,237</td><td align="right">0</td><td align="center">A</td><td align="center">C</td><td align="right">31.5%
</td><td align="center">308.8
/&nbsp;22.0
</td><td align="center">130</td><td align="center">intergenic&nbsp;(&#8209;552/+365)</td><td align="center"><i>A1S_3466/A1S_3467</i></td><td align="left">Resolvase/hypothetical protein</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_53.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">3,121,854</td><td align="right">0</td><td align="center">A</td><td align="center">C</td><td align="right">30.2%
</td><td align="center">285.9
/&nbsp;19.0
</td><td align="center">120</td><td align="center"><font class="snp_type_nonsynonymous">Y148S</font>&nbsp;(T<font class="mutation_in_codon">A</font>T&rarr;T<font class="mutation_in_codon">C</font>T)&nbsp;</td><td align="center"><i>tcuB</i></td><td align="left">tricarballylate utilization protein B</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_28.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">951,186</td><td align="right">0</td><td align="center">A</td><td align="center">C</td><td align="right">29.9%
</td><td align="center">215.4
/&nbsp;17.3
</td><td align="center">107</td><td align="center"><font class="snp_type_nonsynonymous">I622L</font>&nbsp;(<font class="mutation_in_codon">A</font>TT&rarr;<font class="mutation_in_codon">C</font>TT)&nbsp;</td><td align="center"><i>ACX60_RS04505</i></td><td align="left">DNA internalization&#8209;related competence protein ComEC/Rec2</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_52.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">2,984,858</td><td align="right">0</td><td align="center">G</td><td align="center">C</td><td align="right">29.8%
</td><td align="center">283.5
/&nbsp;71.0
</td><td align="center">125</td><td align="center">noncoding&nbsp;(344/355&nbsp;nt)</td><td align="center"><i>ACX60_RS19000</i></td><td align="left">bacterial RNase P</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_20.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">419,330</td><td align="right">0</td><td align="center">A</td><td align="center">C</td><td align="right">29.5%
</td><td align="center">301.0
/&nbsp;29.3
</td><td align="center">130</td><td align="center">intergenic&nbsp;(+55/&#8209;52)</td><td align="center"><i>ACX60_RS01970/ACX60_RS01975</i></td><td align="left">DEAD/DEAH box helicase/hypothetical protein</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_37.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">1,847,596</td><td align="right">0</td><td align="center">G</td><td align="center">C</td><td align="right">29.3%
</td><td align="center">164.3
/&nbsp;32.9
</td><td align="center">78</td><td align="center"><font class="snp_type_nonsynonymous">P121R</font>&nbsp;(C<font class="mutation_in_codon">C</font>C&rarr;C<font class="mutation_in_codon">G</font>C)&nbsp;</td><td align="center"><i>ACX60_RS08700</i></td><td align="left">LysR family transcriptional regulator</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_48.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">2,656,953</td><td align="right">0</td><td align="center">T</td><td align="center">G</td><td align="right">28.7%
</td><td align="center">186.7
/&nbsp;15.2
</td><td align="center">98</td><td align="center"><font class="snp_type_nonsense">L180*</font>&nbsp;(T<font class="mutation_in_codon">T</font>A&rarr;T<font class="mutation_in_codon">G</font>A)&nbsp;</td><td align="center"><i>ACX60_RS12725</i></td><td align="left">transferase</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_24.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">682,098</td><td align="right">0</td><td align="center">A</td><td align="center">C</td><td align="right">28.3%
</td><td align="center">293.1
/&nbsp;10.6
</td><td align="center">119</td><td align="center">intergenic&nbsp;(+62/+63)</td><td align="center"><i>ACX60_RS03295/ACX60_RS03300</i></td><td align="left">membrane protein/hypothetical protein</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_44.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">2,204,396</td><td align="right">0</td><td align="center">C</td><td align="center">G</td><td align="right">27.5%
</td><td align="center">105.6
/&nbsp;22.0
</td><td align="center">55</td><td align="center">intergenic&nbsp;(&#8209;8/&#8209;251)</td><td align="center"><i>ACX60_RS10510/ACX60_RS10515</i></td><td align="left">hypothetical protein/hypothetical protein</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_29.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">984,391</td><td align="right">0</td><td align="center">T</td><td align="center">G</td><td align="right">27.4%
</td><td align="center">293.3
/&nbsp;14.1
</td><td align="center">97</td><td align="center"><font class="snp_type_nonsynonymous">K297Q</font>&nbsp;(<font class="mutation_in_codon">A</font>AG&rarr;<font class="mutation_in_codon">C</font>AG)&nbsp;</td><td align="center"><i>ACX60_RS04655</i></td><td align="left">AraC family transcriptional regulator</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_46.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">2,349,705</td><td align="right">0</td><td align="center">T</td><td align="center">C</td><td align="right">27.0%
</td><td align="center">179.0
/&nbsp;28.6
</td><td align="center">100</td><td align="center">intergenic&nbsp;(+101/&#8209;5)</td><td align="center"><i>ACX60_RS11215/ACX60_RS11220</i></td><td align="left">hypothetical protein/hypothetical protein</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_41.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">2,095,886</td><td align="right">0</td><td align="center">C</td><td align="center">G</td><td align="right">26.8%
</td><td align="center">153.0
/&nbsp;30.9
</td><td align="center">82</td><td align="center"><font class="snp_type_nonsynonymous">S453T</font>&nbsp;(A<font class="mutation_in_codon">G</font>T&rarr;A<font class="mutation_in_codon">C</font>T)&nbsp;</td><td align="center"><i>ACX60_RS09900</i></td><td align="left">heme&#8209;binding protein A</td></tr>
</table>

<p>
<table border="0" cellspacing="1" cellpadding="3">
<tr><th colspan="12" align="left" class="new_junction_header_row">Marginal new junction evidence (sorted from low to high skew)</th></tr>
<!-- Header Lines for New Junction -->
<tr>
<th colspan="2">&nbsp;</th>
<th>seq&nbsp;id</th>
<th>position</th>
<th>reads&nbsp;(cov)</th>
<th>reads&nbsp;(cov)</th>
<th>score</th>
<th>skew</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">product</th>
</tr>

<!-- Item Lines for New Junction -->
<!-- Side 1 Item Lines for New Junction -->
<tr class="mutation_table_row_0">
<td rowspan="2"><a href="evidence/JC_66.html">*</a></td>
<td rowspan="1"><a href="evidence/JC_SIDE_1_66.html">?</a></td>
<td rowspan="1" class="junction_gene">NZ_CP012004</td>
<td align="center" class="junction_gene">131179&nbsp;=</td><td align="center" class="junction_gene">139 (1.090)</td><td rowspan="2" align="center">4 (0.040)</td>
<td rowspan="2" align="center">3/248</td>
<td rowspan="2" align="center">5.4</td>
<td rowspan="2" align="center">3.1%</td>
<td align="center" class="junction_gene">coding&nbsp;(106/183&nbsp;nt)</td>
<td align="center" class="junction_gene"><i>ACX60_RS00600</i></td>
<td class="junction_gene">hypothetical protein</td>
</tr>
<!-- Side 2 Item Lines for New Junction -->
<tr class="mutation_table_row_0">
<td rowspan="1"><a href="evidence/JC_SIDE_2_66.html">?</a></td><td rowspan="1" class="junction_gene">NZ_CP012004</td>
<td align="center" class="junction_gene">=&nbsp;131175</td>
<td align="center" class="junction_gene">135 (1.250)</td><td align="center" class="junction_gene">coding&nbsp;(110/183&nbsp;nt)</td>
<td align="center" class="junction_gene"><i>ACX60_RS00600</i></td>
<td class="junction_gene">hypothetical protein</td>
</tr>
</table>
</body></html>