<!DOCTYPE html
PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
<html>
<head>
<title>BRESEQ :: Summary Statistics</title>
<style type = "text/css">
body {font-family: sans-serif; font-size: 11pt;}
th {background-color: rgb(0,0,0); color: rgb(255,255,255);}
table {background-color: rgb(1,0,0); color: rgb(0,0,0);}
tr {background-color: rgb(255,255,255);}
.mutation_in_codon {color:red; text-decoration : underline;}
.snp_type_synonymous{color:green;}
.snp_type_nonsynonymous{color:blue;}
.snp_type_nonsense{color:red;}
.mutation_header_row {background-color: rgb(0,130,0);}
.read_alignment_header_row {background-color: rgb(255,0,0);}
.missing_coverage_header_row {background-color: rgb(0,100,100);}
.new_junction_header_row {background-color: rgb(0,0,155);}
.copy_number_header_row {background-color: rgb(153,102,0);}
.alternate_table_row_0 {background-color: rgb(255,255,255);}
.alternate_table_row_1 {background-color: rgb(235,235,235);}
.gray_table_row {background-color: rgb(230,230,245);}
.polymorphism_table_row {background-color: rgb(160,255,160);}
.highlight_table_row {background-color: rgb(192,255,255);}
.reject_table_row {background-color: rgb(255,200,165);}
.user_defined_table_row {background-color: rgb(255,255,0);}
.information_table_row {background-color: rgb(200,255,255);}
.junction_repeat {background-color: rgb(255,165,0)}
.junction_gene {}
.hidden { display: none; }
.unhidden { display: block; }

</style>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1" />
<script type="text/javascript">
  function hideTog(divID) {
    var item = document.getElementById(divID);
    if (item) {
      item.className=(item.className=='hidden')?'unhidden':'hidden';
    }
  }
  function showTog(butID) {
    var button = document.getElementById(butID);
    if (button) {
      button.value=(button.value=='Show')?'Hide':'Show';
    }
  }
</script>

</head>
<body>
<table width="100%" border="0" cellspacing="0" cellpadding="3">
<tr>
<td><a href="http://barricklab.org/breseq"><img src="evidence/breseq_small.png" /></a></td>
<td width="100%">
<b><i>breseq</i></b>&nbsp;&nbsp;version 0.38.1&nbsp;&nbsp;
<br><a href="index.html">mutation predictions</a> | 
<a href="marginal.html">marginal predictions</a> | 
<a href="summary.html">summary statistics</a> | 
<a href="output.gd">genome diff</a> | 
<a href="log.txt">command line log</a>
</td></tr></table>

<p>
<h2>Read File Information</h2>
<table border="0" cellspace="1" cellpadding="5">
<tr ><th></th><th>read file</th><th>reads</th><th>bases</th><th>passed&nbsp;filters</th><th>average</th><th>longest</th><th>mapped</th></tr>
<tr ><td><a href="calibration/YDA99_S80_R1_001.error_rates.pdf">errors</a></td><td>YDA99_S80_R1_001</td><td align="right">1,715,946</td><td align="right">252,649,460</td><td align="right">100.0%</td><td align="right">147.2&nbsp;bases</td><td align="right">151&nbsp;bases</td><td align="right">98.3%</td></tr><tr ><td><a href="calibration/YDA99_S80_R2_001.error_rates.pdf">errors</a></td><td>YDA99_S80_R2_001</td><td align="right">1,703,687</td><td align="right">250,793,136</td><td align="right">99.3%</td><td align="right">147.2&nbsp;bases</td><td align="right">151&nbsp;bases</td><td align="right">97.7%</td></tr><tr class="highlight_table_row"><td></td><td><b>total</b></td><td align="right"><b>3,419,633</b></td><td align="right"><b>503,442,596</b></td><td align="right">99.6%</td><td align="right">147.2&nbsp;bases</td><td align="right"><b>151</b>&nbsp;bases</td><td align="right">98.0%</td></tr></table><h2>Reference Sequence Information</h2>
<p>
<table border="0" cellspacing="1" cellpadding="5" >
<tr><th></th><th></th><th>seq id</th><th>length</th><th align="center">fit mean</th><th align="center">fit relative_variance</th><th align="center">% mapped reads</th><th align="left">description</th></tr>
<tr><td><a href="evidence/NZ_CP012004.overview.png">coverage</a></td><td><a href="calibration/0.unique_coverage.pdf">distribution</a></td><td>NZ_CP012004</td><td align="right">3,857,738</td><td align="center">127.4</td><td align="center">11.6</td><td align="center">98.1%</td><td>Acinetobacter baumannii ATCC 17978-mff, complete genome.</td></tr><tr><td><a href="evidence/CP000522.overview.png">coverage</a></td><td><a href="calibration/1.unique_coverage.pdf">distribution</a></td><td>CP000522</td><td align="right">13,408</td><td align="center">128.6</td><td align="center">3.9</td><td align="center">0.4%</td><td>Acinetobacter baumannii ATCC 17978 plasmid pAB1, complete sequence.</td></tr><tr><td><a href="evidence/CP000523.overview.png">coverage</a></td><td><a href="calibration/2.unique_coverage.pdf">distribution</a></td><td>CP000523</td><td align="right">11,302</td><td align="center">363.2</td><td align="center">7.6</td><td align="center">0.8%</td><td>Acinetobacter baumannii ATCC 17978 plasmid pAB2, complete sequence.</td></tr><tr><td><a href="evidence/CP012005.overview.png">coverage</a></td><td><a href="calibration/3.unique_coverage.pdf">distribution</a></td><td>CP012005</td><td align="right">148,955</td><td align="center">17.9</td><td align="center">1.8</td><td align="center">0.7%</td><td>Acinetobacter baumannii ATCC 17978-mff plasmid pAB3, complete</td></tr><tr class="highlight_table_row"><td></td><td></td><td><b>total</b></td><td align="right"><b>4,031,403</b></td><td></td><td></td><td align="center">100.0%</td><td></td></tr>
</table>
<p><b>fit relative_variance</b> is the ratio of the variance to the mean for the negative binomial fit. It is =1 for Poisson and >1 for over-dispersed data.
<h2>New Junction Evidence</h2>
<p>
<h3>Junction Candidates Tested</h3>
<p>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th align="center">option</th><th align="center">limit</th><th align="center">actual</th></tr><tr><td>Number of alignment pairs examined for constructing junction candidates</td><td align="center">&le;&nbsp;100000</td><td align="center">1338</td></tr><tr><td>Coverage evenness (position-hash) score of junction candidates</td><td align="center">&ge;&nbsp;2</td><td align="center">&ge;&nbsp;2</td></tr><tr><td>Test this many junction candidates (n). May be smaller if not enough passed the coverage evenness threshold</td><td align="center">100&nbsp;&le;&nbsp;n&nbsp;&le;&nbsp;5000</td><td align="center">32</td></tr><tr><td>Total length of all junction candidates (factor times the reference genome length)</td><td align="center">&le;&nbsp;0.1</td><td align="center">0.003</td></tr></table>
<h3>Junction Skew Score Calculation</h3>
<p>
<table border="0" cellspacing="1" cellpadding="5">
<tr ><th>reference sequence</th><th>pr(no read start)</th></tr>
<tr >
<td>NZ_CP012004</td><td align="center">0.82848</td></tr><tr >
<td>CP000522</td><td align="center">0.82924</td></tr><tr >
<td>CP000523</td><td align="center">0.71974</td></tr><tr >
<td>CP012005</td><td align="center">0.95702</td></tr></table><p><b>pr(no read start)</b> is the probability that there will not be an aligned read whose first base matches a given position on a given strand.
<h3>Final Junction Predictions</h3>
<p>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th align="center">option</th><th align="center">value</th></tr><tr><td>Coverage evenness (position-hash) score of predicted junctions must be</td><td align="center">&ge;&nbsp;3</td></tr><tr><td>Minimum probablilty assigned that no mapped read will start at a given position and strand for junction prediction</td><td align="center">0.1</td></tr><tr><td>Junction allow suboptimal matches</td><td align="center">FALSE</td></tr><tr><td>Skew score of predicted junction (&minus;log10 probability of unusual coverage evenness) must be</td><td align="center">&le;&nbsp;3</td></tr><tr><td>Number of bases that at least one read must overlap each uniquely aligned side of a predicted junction</td><td align="center">&ge;&nbsp;1</td></tr></table>
<h2>Read Alignment Evidence</h2>
<p>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th>option</th><th>value</th></tr><tr><td>Mode</td><td>Consensus/Mixed Base</td></tr><tr><td>Ploidy</td><td>1 (haploid)</td></tr><tr><td>Consensus mutation E-value cutoff</td><td>10</td></tr><tr><td>Consensus frequency cutoff</td><td>0.8</td></tr><tr><td>Consensus minimum variant coverage each strand</td><td>OFF</td></tr><tr><td>Consensus minimum total coverage each strand</td><td>OFF</td></tr><tr><td>Consensus minimum variant coverage</td><td>OFF</td></tr><tr><td>Consensus minimum total coverage</td><td>OFF</td></tr><tr><td>Polymorphism E-value cutoff</td><td>10</td></tr><tr><td>Polymorphism frequency cutoff</td><td>0.2</td></tr><tr><td>Polymorphism minimum variant coverage each strand</td><td>OFF</td></tr><tr><td>Polymorphism minimum total coverage each strand</td><td>OFF</td></tr><tr><td>Polymorphism minimum variant coverage</td><td>OFF</td></tr><tr><td>Polymorphism minimum total coverage</td><td>OFF</td></tr><tr><td>Polymorphism bias cutoff</td><td>OFF</td></tr><tr><td>Predict indel polymorphisms</td><td>YES</td></tr><tr><td>Skip indel polymorphisms in homopolymers runs of</td><td>OFF</td></tr><tr><td>Skip base substitutions when they create a homopolymer flanked on each side by</td><td>OFF</td></tr></table><p>
<h2>Software Versions</h2>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th>program</th><th>version</th></tr>
<tr><td>bowtie2</td><td>2.5.2</td></tr>
<tr><td>R</td><td>4.3.1</td></tr>
</table><p>
<h2>Execution Times</h2>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th>step</th><th>start</th><th>end</th><th>elapsed</th></tr>
<tr><td>Read and reference sequence file input</td><td>09:41:15&nbsp;30&nbsp;May&nbsp;2024</td><td>09:41:51&nbsp;30&nbsp;May&nbsp;2024</td><td>36&nbsp;seconds</td></tr>
<tr><td>Read alignment to reference genome</td><td>09:41:51&nbsp;30&nbsp;May&nbsp;2024</td><td>10:02:10&nbsp;30&nbsp;May&nbsp;2024</td><td>20&nbsp;minutes&nbsp;19&nbsp;seconds</td></tr>
<tr><td>Preprocessing alignments for candidate junction identification</td><td>10:02:10&nbsp;30&nbsp;May&nbsp;2024</td><td>10:03:00&nbsp;30&nbsp;May&nbsp;2024</td><td>50&nbsp;seconds</td></tr>
<tr><td>Preliminary analysis of coverage distribution</td><td>10:03:00&nbsp;30&nbsp;May&nbsp;2024</td><td>10:04:39&nbsp;30&nbsp;May&nbsp;2024</td><td>1&nbsp;minute&nbsp;39&nbsp;seconds</td></tr>
<tr><td>Identifying junction candidates</td><td>10:04:39&nbsp;30&nbsp;May&nbsp;2024</td><td>10:04:39&nbsp;30&nbsp;May&nbsp;2024</td><td>0&nbsp;seconds</td></tr>
<tr><td>Re-alignment to junction candidates</td><td>10:04:39&nbsp;30&nbsp;May&nbsp;2024</td><td>10:18:12&nbsp;30&nbsp;May&nbsp;2024</td><td>13&nbsp;minutes&nbsp;33&nbsp;seconds</td></tr>
<tr><td>Resolving best read alignments</td><td>10:18:12&nbsp;30&nbsp;May&nbsp;2024</td><td>10:19:30&nbsp;30&nbsp;May&nbsp;2024</td><td>1&nbsp;minute&nbsp;18&nbsp;seconds</td></tr>
<tr><td>Creating BAM files</td><td>10:19:30&nbsp;30&nbsp;May&nbsp;2024</td><td>10:20:52&nbsp;30&nbsp;May&nbsp;2024</td><td>1&nbsp;minute&nbsp;22&nbsp;seconds</td></tr>
<tr><td>Tabulating error counts</td><td>10:20:52&nbsp;30&nbsp;May&nbsp;2024</td><td>10:22:03&nbsp;30&nbsp;May&nbsp;2024</td><td>1&nbsp;minute&nbsp;11&nbsp;seconds</td></tr>
<tr><td>Re-calibrating base error rates</td><td>10:22:03&nbsp;30&nbsp;May&nbsp;2024</td><td>10:22:07&nbsp;30&nbsp;May&nbsp;2024</td><td>4&nbsp;seconds</td></tr>
<tr><td>Examining read alignment evidence</td><td>10:22:07&nbsp;30&nbsp;May&nbsp;2024</td><td>10:31:05&nbsp;30&nbsp;May&nbsp;2024</td><td>8&nbsp;minutes&nbsp;58&nbsp;seconds</td></tr>
<tr><td>Polymorphism statistics</td><td>10:31:05&nbsp;30&nbsp;May&nbsp;2024</td><td>10:31:05&nbsp;30&nbsp;May&nbsp;2024</td><td>0&nbsp;seconds</td></tr>
<tr><td>Output</td><td>10:31:05&nbsp;30&nbsp;May&nbsp;2024</td><td>10:31:23&nbsp;30&nbsp;May&nbsp;2024</td><td>18&nbsp;seconds</td></tr>
<tr><td>Output :: Mutation Prediction</td><td>10:31:05&nbsp;30&nbsp;May&nbsp;2024</td><td>10:31:06&nbsp;30&nbsp;May&nbsp;2024</td><td>1&nbsp;second</td></tr>
<tr><td>Output :: Mutation Annotation</td><td>10:31:06&nbsp;30&nbsp;May&nbsp;2024</td><td>10:31:06&nbsp;30&nbsp;May&nbsp;2024</td><td>0&nbsp;seconds</td></tr>
<tr class="highlight_table_row">
<td colspan="3" ><b>Total</b></td>
<td><b>50&nbsp;minutes&nbsp;9&nbsp;seconds</b></td>
</tr>
</table></body></html>