<!DOCTYPE html
PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
<html>
<head>
<title>BRESEQ :: Marginal Predictions</title>
<style type = "text/css">
body {font-family: sans-serif; font-size: 11pt;}
th {background-color: rgb(0,0,0); color: rgb(255,255,255);}
table {background-color: rgb(1,0,0); color: rgb(0,0,0);}
tr {background-color: rgb(255,255,255);}
.mutation_in_codon {color:red; text-decoration : underline;}
.snp_type_synonymous{color:green;}
.snp_type_nonsynonymous{color:blue;}
.snp_type_nonsense{color:red;}
.mutation_header_row {background-color: rgb(0,130,0);}
.read_alignment_header_row {background-color: rgb(255,0,0);}
.missing_coverage_header_row {background-color: rgb(0,100,100);}
.new_junction_header_row {background-color: rgb(0,0,155);}
.copy_number_header_row {background-color: rgb(153,102,0);}
.alternate_table_row_0 {background-color: rgb(255,255,255);}
.alternate_table_row_1 {background-color: rgb(235,235,235);}
.gray_table_row {background-color: rgb(230,230,245);}
.polymorphism_table_row {background-color: rgb(160,255,160);}
.highlight_table_row {background-color: rgb(192,255,255);}
.reject_table_row {background-color: rgb(255,200,165);}
.user_defined_table_row {background-color: rgb(255,255,0);}
.information_table_row {background-color: rgb(200,255,255);}
.junction_repeat {background-color: rgb(255,165,0)}
.junction_gene {}
.hidden { display: none; }
.unhidden { display: block; }

</style>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1" />
<script type="text/javascript">
  function hideTog(divID) {
    var item = document.getElementById(divID);
    if (item) {
      item.className=(item.className=='hidden')?'unhidden':'hidden';
    }
  }
  function showTog(butID) {
    var button = document.getElementById(butID);
    if (button) {
      button.value=(button.value=='Show')?'Hide':'Show';
    }
  }
</script>

</head>
<body>
<table width="100%" border="0" cellspacing="0" cellpadding="3">
<tr>
<td><a href="http://barricklab.org/breseq"><img src="evidence/breseq_small.png" /></a></td>
<td width="100%">
<b><i>breseq</i></b>&nbsp;&nbsp;version 0.38.1&nbsp;&nbsp;
<br><a href="index.html">mutation predictions</a> | 
<a href="marginal.html">marginal predictions</a> | 
<a href="summary.html">summary statistics</a> | 
<a href="output.gd">genome diff</a> | 
<a href="log.txt">command line log</a>
</td></tr></table>

<p>
<p>
<table border="0" cellspacing="1", cellpadding="3">
<tr><th colspan="12" align="left" class="read_alignment_header_row">Marginal read alignment evidence (highest frequency 20 of 48 shown, sorted by frequency from high to low)</th></tr>
<tr>
<th>&nbsp;</th>
<th>seq&nbsp;id</th>
<th colspan="2">position</th>
<th>ref</th>
<th>new</th>
<th>freq</th>
<th>score&nbsp;(cons/poly)</th>
<th>reads</th>
<th>annotation</th>
<th>genes</th>
<th width="100%">product</th>
</tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_51.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">2,736,592</td><td align="right">0</td><td align="center">C</td><td align="center">G</td><td align="right">47.7%
</td><td align="center">88.8
/&nbsp;119.5
</td><td align="center">114</td><td align="center"><font class="snp_type_nonsynonymous">S479T</font>&nbsp;(A<font class="mutation_in_codon">G</font>T&rarr;A<font class="mutation_in_codon">C</font>T)&nbsp;</td><td align="center"><i>fadH</i></td><td align="left">NADPH&#8209;dependent 2,4&#8209;dienoyl&#8209;CoA reductase</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_31.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">1,611,384</td><td align="right">0</td><td align="center">G</td><td align="center">C</td><td align="right">37.2%
</td><td align="center">196.6
/&nbsp;130.3
</td><td align="center">162</td><td align="center"><font class="snp_type_nonsynonymous">V152L</font>&nbsp;(<font class="mutation_in_codon">G</font>TC&rarr;<font class="mutation_in_codon">C</font>TC)&nbsp;</td><td align="center"><i>ACX60_RS07530</i></td><td align="left">hypothetical protein</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_29.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">1,529,192</td><td align="right">0</td><td align="center">A</td><td align="center">C</td><td align="right">35.6%
</td><td align="center">224.1
/&nbsp;17.7
</td><td align="center">118</td><td align="center">intergenic&nbsp;(+51/&#8209;588)</td><td align="center"><i>ACX60_RS07100/ACX60_RS07105</i></td><td align="left">aminopeptidase N/spore Coat protein U domain family</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_56.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">3,125,709</td><td align="right">0</td><td align="center">C</td><td align="center">G</td><td align="right">35.2%
</td><td align="center">266.5
/&nbsp;94.6
</td><td align="center">147</td><td align="center"><font class="snp_type_nonsynonymous">G330R</font>&nbsp;(<font class="mutation_in_codon">G</font>GC&rarr;<font class="mutation_in_codon">C</font>GC)&nbsp;</td><td align="center"><i>ACX60_RS14780</i></td><td align="left">acyl&#8209;CoA dehydrogenase</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_48.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">2,656,953</td><td align="right">0</td><td align="center">T</td><td align="center">G</td><td align="right">34.1%
</td><td align="center">253.8
/&nbsp;18.9
</td><td align="center">133</td><td align="center"><font class="snp_type_nonsense">L180*</font>&nbsp;(T<font class="mutation_in_codon">T</font>A&rarr;T<font class="mutation_in_codon">G</font>A)&nbsp;</td><td align="center"><i>ACX60_RS12725</i></td><td align="left">transferase</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_40.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">2,204,396</td><td align="right">0</td><td align="center">C</td><td align="center">G</td><td align="right">33.3%
</td><td align="center">154.2
/&nbsp;49.6
</td><td align="center">86</td><td align="center">intergenic&nbsp;(&#8209;8/&#8209;251)</td><td align="center"><i>ACX60_RS10510/ACX60_RS10515</i></td><td align="left">hypothetical protein/hypothetical protein</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_54.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">2,984,858</td><td align="right">0</td><td align="center">G</td><td align="center">C</td><td align="right">32.9%
</td><td align="center">354.9
/&nbsp;88.4
</td><td align="center">161</td><td align="center">noncoding&nbsp;(344/355&nbsp;nt)</td><td align="center"><i>ACX60_RS19000</i></td><td align="left">bacterial RNase P</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_27.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">951,186</td><td align="right">0</td><td align="center">A</td><td align="center">C</td><td align="right">32.6%
</td><td align="center">264.5
/&nbsp;19.0
</td><td align="center">137</td><td align="center"><font class="snp_type_nonsynonymous">I622L</font>&nbsp;(<font class="mutation_in_codon">A</font>TT&rarr;<font class="mutation_in_codon">C</font>TT)&nbsp;</td><td align="center"><i>ACX60_RS04505</i></td><td align="left">DNA internalization&#8209;related competence protein ComEC/Rec2</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_35.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">1,939,477</td><td align="right">0</td><td align="center">T</td><td align="center">A</td><td align="right">31.6%
</td><td align="center">32.9
/&nbsp;21.7
</td><td align="center">19</td><td align="center">pseudogene&nbsp;(6/369&nbsp;nt)</td><td align="center"><i>antC</i></td><td align="left">anthranilate dioxygenase reductase</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_38.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">2,204,394</td><td align="right">0</td><td align="center">A</td><td align="center">G</td><td align="right">31.5%
</td><td align="center">148.0
/&nbsp;55.0
</td><td align="center">92</td><td align="center">intergenic&nbsp;(&#8209;6/&#8209;253)</td><td align="center"><i>ACX60_RS10510/ACX60_RS10515</i></td><td align="left">hypothetical protein/hypothetical protein</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_39.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">2,204,395</td><td align="right">0</td><td align="center">C</td><td align="center">G</td><td align="right">30.9%
</td><td align="center">173.5
/&nbsp;44.7
</td><td align="center">88</td><td align="center">intergenic&nbsp;(&#8209;7/&#8209;252)</td><td align="center"><i>ACX60_RS10510/ACX60_RS10515</i></td><td align="left">hypothetical protein/hypothetical protein</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_41.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">2,204,397</td><td align="right">0</td><td align="center">C</td><td align="center">G</td><td align="right">30.7%
</td><td align="center">167.4
/&nbsp;38.0
</td><td align="center">86</td><td align="center">intergenic&nbsp;(&#8209;9/&#8209;250)</td><td align="center"><i>ACX60_RS10510/ACX60_RS10515</i></td><td align="left">hypothetical protein/hypothetical protein</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_22.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">682,086</td><td align="right">0</td><td align="center">A</td><td align="center">C</td><td align="right">30.5%
</td><td align="center">230.2
/&nbsp;19.7
</td><td align="center">131</td><td align="center">intergenic&nbsp;(+50/+75)</td><td align="center"><i>ACX60_RS03295/ACX60_RS03300</i></td><td align="left">membrane protein/hypothetical protein</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_19.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">419,330</td><td align="right">0</td><td align="center">A</td><td align="center">C</td><td align="right">30.3%
</td><td align="center">252.0
/&nbsp;12.8
</td><td align="center">100</td><td align="center">intergenic&nbsp;(+55/&#8209;52)</td><td align="center"><i>ACX60_RS01970/ACX60_RS01975</i></td><td align="left">DEAD/DEAH box helicase/hypothetical protein</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_60.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">3,632,436</td><td align="right">0</td><td align="center">A</td><td align="center">C</td><td align="right">30.3%
</td><td align="center">283.4
/&nbsp;19.7
</td><td align="center">133</td><td align="center"><font class="snp_type_nonsynonymous">L70F</font>&nbsp;(TT<font class="mutation_in_codon">A</font>&rarr;TT<font class="mutation_in_codon">C</font>)&nbsp;</td><td align="center"><i>ACX60_RS17130</i></td><td align="left">MFS transporter</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_58.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">3,404,409</td><td align="right">0</td><td align="center">T</td><td align="center">G</td><td align="right">29.5%
</td><td align="center">409.6
/&nbsp;18.9
</td><td align="center">156</td><td align="center"><font class="snp_type_nonsynonymous">L138F</font>&nbsp;(TT<font class="mutation_in_codon">A</font>&rarr;TT<font class="mutation_in_codon">C</font>)&nbsp;</td><td align="center"><i>ACX60_RS16090</i></td><td align="left">23S rRNA pseudouridine synthase</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_28.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">1,145,469</td><td align="right">0</td><td align="center">G</td><td align="center">C</td><td align="right">29.5%
</td><td align="center">245.5
/&nbsp;68.8
</td><td align="center">160</td><td align="center">intergenic&nbsp;(+37/+11)</td><td align="center"><i>ACX60_RS05360/ACX60_RS05365</i></td><td align="left">hypothetical protein/membrane protein</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_34.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">1,847,597</td><td align="right">0</td><td align="center">G</td><td align="center">C</td><td align="right">29.3%
</td><td align="center">248.4
/&nbsp;46.1
</td><td align="center">111</td><td align="center"><font class="snp_type_nonsynonymous">P121A</font>&nbsp;(<font class="mutation_in_codon">C</font>CC&rarr;<font class="mutation_in_codon">G</font>CC)&nbsp;&Dagger;</td><td align="center"><i>ACX60_RS08700</i></td><td align="left">LysR family transcriptional regulator</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_33.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">1,847,596</td><td align="right">0</td><td align="center">G</td><td align="center">C</td><td align="right">29.2%
</td><td align="center">247.7
/&nbsp;54.6
</td><td align="center">111</td><td align="center"><font class="snp_type_nonsynonymous">P121R</font>&nbsp;(C<font class="mutation_in_codon">C</font>C&rarr;C<font class="mutation_in_codon">G</font>C)&nbsp;&Dagger;</td><td align="center"><i>ACX60_RS08700</i></td><td align="left">LysR family transcriptional regulator</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_49.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">2,668,823</td><td align="right">0</td><td align="center">T</td><td align="center">G</td><td align="right">28.7%
</td><td align="center">252.1
/&nbsp;21.8
</td><td align="center">115</td><td align="center"><font class="snp_type_nonsynonymous">L216F</font>&nbsp;(TT<font class="mutation_in_codon">A</font>&rarr;TT<font class="mutation_in_codon">C</font>)&nbsp;</td><td align="center"><i>ACX60_RS12775</i></td><td align="left">GntR family transcriptional regulator</td></tr>
</table>

<p>
<table border="0" cellspacing="1" cellpadding="3">
<tr><th colspan="12" align="left" class="new_junction_header_row">Marginal new junction evidence (sorted from low to high skew)</th></tr>
<!-- Header Lines for New Junction -->
<tr>
<th colspan="2">&nbsp;</th>
<th>seq&nbsp;id</th>
<th>position</th>
<th>reads&nbsp;(cov)</th>
<th>reads&nbsp;(cov)</th>
<th>score</th>
<th>skew</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">product</th>
</tr>

<!-- Item Lines for New Junction -->
<!-- Side 1 Item Lines for New Junction -->
<tr class="mutation_table_row_0">
<td rowspan="2"><a href="evidence/JC_72.html">*</a></td>
<td rowspan="1"><a href="evidence/JC_SIDE_1_72.html">?</a></td>
<td rowspan="1" class="junction_gene">NZ_CP012004</td>
<td align="center" class="junction_gene">1591231&nbsp;=</td><td align="center" class="junction_gene">161 (1.150)</td><td rowspan="2" align="center">4 (0.030)</td>
<td rowspan="2" align="center">4/254</td>
<td rowspan="2" align="center">10.7</td>
<td rowspan="2" align="center">2.9%</td>
<td align="center" class="junction_gene">intergenic&nbsp;(+321/&#8209;178)</td>
<td align="center" class="junction_gene"><i>ACX60_RS07370/ACX60_RS07375</i></td>
<td class="junction_gene">hypothetical protein/phage integrase</td>
</tr>
<!-- Side 2 Item Lines for New Junction -->
<tr class="mutation_table_row_0">
<td rowspan="1"><a href="evidence/JC_SIDE_2_72.html">?</a></td><td rowspan="1" class="junction_gene">NZ_CP012004</td>
<td align="center" class="junction_gene">=&nbsp;1642100</td>
<td align="center" class="junction_gene">131 (1.070)</td><td align="center" class="junction_gene">intergenic&nbsp;(&#8209;550/&#8209;40)</td>
<td align="center" class="junction_gene"><i>ACX60_RS07710/ACX60_RS07715</i></td>
<td class="junction_gene">hypothetical protein/MFS transporter</td>
</tr>
</table>
</body></html>